bigtable/examples/README.md

```diff
@@ -0,0 +1,56 @@
+# Cloud Bigtable C++ Client Quick Start Code Samples
+
+This directory contains minimal examples to get started quickly, and are
+referenced from the Doxygen landing page as as "Hello World".
+
+## Compiling the Quickstart Examples
+
+These examples are compiled as part of the build for the Cloud Bigtable C++
+Client.
+
+## Creating a Cloud Bigtable Instance
+
+These examples can connect to a production instance of Cloud Bigtable or to the
+Cloud Bigtable Emulator.  To use use a production instance, please consult the
+Cloud Bigtable
+[documentation](https://cloud.google.com/bigtable/docs/creating-instance).
+To use the Cloud Bigtable emulator, please consult its
```

Comment:
"please consult its" typo?

Reply:
I think that is correct:

https://en.oxforddictionaries.com/usage/its-or-it-s

"The documentation belonging to the Cloud Bigtable emulator".  Changed it anyway.

Reply:
My Bad. Didn't see documentation.